include/query.php

```diff
@@ -12,12 +12,12 @@ class PLL_Query {
 	/**
 	 * @var PLL_Model
 	 */
-	protected $model;
+	public $model;
```

Comment:
I'm not sure if this is useful for the `$model` property but okay for me.

Reply:
> I'm not sure if this is useful for the `$model` property but okay for me.

I asked myself the same question beacause it seems that $model property isn't accessed externaly.

Reply:
I am not sure either. But I did not want to search. Maybe some day, we will take the time to look at all properties that are public and should not.